tests/ci/env_helper.py

```diff
@@ -46,6 +46,12 @@
 )
 CI_CONFIG_PATH = f"{TEMP_PATH}/ci_config.json"
 
+CLICKHOUSE_TEST_STAT_LOGIN = os.getenv("CLICKHOUSE_TEST_STAT_LOGIN")
+CLICKHOUSE_TEST_STAT_PASSWORD = os.getenv("CLICKHOUSE_TEST_STAT_PASSWORD")
+CLICKHOUSE_TEST_STAT_URL = os.getenv("CLICKHOUSE_TEST_STAT_URL")
+DOCKER_PASSWORD = os.getenv("DOCKER_PASSWORD")
+ROBOT_TOKEN = os.getenv("ROBOT_TOKEN")
+
```

Comment:
Are those coming from github secrets also? We need to make sure those are not leaked with logs too